### PR TITLE
Added yaml file for OrderForm

### PR DESCRIPTION
**Pull request by spartanbear**
Added yaml file for OrderForm for both Dev and Test

### CODE REVIEW

File: Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          value: "orderform"
+          ##Public browse base URL
+        - name: PUBLIC_BROWSE_BASE_URL
+          value: "https://buyingcataloguedev.uksouth.cloudapp.azure.com"
+          ##Environment
+        - name: NODE_ENV
+          value: "development"
+          ##The base uri the app is running on to pass to identity service for redirection
+        - name: APP_BASE_URI
+          value: "https://buyingcataloguetestpublic.uksouth.cloudapp.azure.com"
```
Comment (stuart-bass-cgi) [inline Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml]: shouldn't this be dev?

File: Kubernetes/OrderForm/test-orderform-kubernetes-manifest.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        env:
+          ##Base URL
+        - name: BASE_URL
+          value: "orderform"
+          ##Public browse base URL
+        - name: PUBLIC_BROWSE_BASE_URL
+          value: "https://buyingcataloguetestpublic.uksouth.cloudapp.azure.com"
+          ##Environment
+        - name: NODE_ENV
+          value: "testelopment"
```
Comment (stuart-bass-cgi) [inline Kubernetes/OrderForm/test-orderform-kubernetes-manifest.yml]: I think this variable should be removed

File: Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        - containerPort: 3006
+        resources:
+          requests:
+            cpu: 250m
+          limits:
+            cpu: 500m
+        env:
+          ##Base URL
+        - name: BASE_URL
+          value: "orderform"
```
Comment (stuart-bass-cgi) [inline Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml]: I think this should be /order

**Review by stuart-bass-bjss — CHANGES_REQUESTED** [review]

File: Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        image: gpitfuturesdevacr.azurecr.io/nhsd/buying-catalogue/order-form
+        ports:
+        - containerPort: 3006
+        resources:
+          requests:
+            cpu: 250m
+          limits:
+            cpu: 500m
+        livenessProbe:
+          httpGet:
```
Comment (stuart-bass-cgi) [inline Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml]: order form doesn't have live / ready yet

**Review by stuart-bass-bjss — CHANGES_REQUESTED** [review]

**Review by stuart-bass-bjss — APPROVED** [review]

**Pull request merged**